### DISCUSSION TITLE
Make it possible to delete review comments

### DISCUSSION THREAD

**Discussion by jay** [community/community#39204] [Pull Requests]
### Select Topic Area

Bug

### Body

About 2 years ago we had a user post to the curl project soliciting bitcoin. Some of the comments this user posted could not be deleted by us because they were PR review or suggestion comments. I wrote GitHub who told me at the time it was on their internal feature request list. Today 2 years later it is still possible to make un-deletable comments. For example, [this review](https://github.com/curl/curl/pull/9919#pullrequestreview-1181603558) just now which makes no sense. Every so often we get a nonsense comment that initially makes no sense and can later be modified to something else, ie spam.

GitHub should consider fixing the moderation feature so that we moderate these types of comments without having to contact GitHub and wait for the turnaround.

**Comment by samus-aran** [comment]
Hi @jay 👋 
Thanks for sharing and we appreciate your patience. You can check our [public roadmap](https://github.com/orgs/github/projects/4247) to see the kinds of work we are focusing on for the coming year. Smaller fixes won’t be listed, but are often included when work is happening adjacent to it.
Although it has been a while since we provided an update on this feature request we can only share at this time that we _are_ currently working on improving this for repos’ owners and maintainers. 🙂 So stay tuned and keep an eye on the public roadmap for more info on this! :eyes:
